### PR TITLE
Badges

### PR DESCRIPTION
**Pull request by ggrimaudDTS**
🦡

### CODE REVIEW

File: @utahdts/utah-design-system/css/1-settings/_spacing.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
 */
 
 @use "../1-settings/class-vars";
-
+// 4 8
```
Comment (AustinHaws-Utah) [inline @utahdts/utah-design-system/css/1-settings/_spacing.scss:7]: Is this comment intentional? what is its purpose?

File: @utahdts/utah-design-system/react/components/buttons/IconButton.jsx [hunk 2]
```diff
@@ -21,10 +22,12 @@ import { Tooltip } from '../Tooltip/Tooltip';
  * @param {'small1x' | 'small' | 'medium' | 'large' | 'large1x'} [props.size]
  * @param {string} props.title A title is used for accessibility purposes to describe the button for screen readers
  * @param {string | null} [props.tooltipText]
+ * @param props.children
```
Comment (AustinHaws-Utah) [inline @utahdts/utah-design-system/react/components/buttons/IconButton.jsx:25]: already has `props.children` detailed above? probably doesn't need it again?

File: @utahdts/utah-design-system/css/1-settings/_spacing.scss (hunk 1 above)
Comment (ggrimaudDTS) [reply @utahdts/utah-design-system/css/1-settings/_spacing.scss:7]: Oops

File: @utahdts/utah-design-system/react/components/widgetsIndicators/Badge.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { joinClassNames } from '../../util/joinClassNames';
+
+/**
+ * @param {object} props
+ * @param {import('react').ReactNode} [props.children]
+ * @param {string} [props.className]
+ * @param {string} props.title
+ * @returns {import('react').JSX.Element}
```
Comment (AustinHaws-Utah) [inline @utahdts/utah-design-system/react/components/widgetsIndicators/Badge.jsx:8]: what other common props should this have from PROPTYPES.md?
- innerRef?
- ...rest?

File: @utahdts/utah-design-system/react/components/widgetsIndicators/Badge.jsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+ */
+export function Badge({
+  children,
+  className,
+  title,
+}) {
+  return (
+    <div className={joinClassNames('badge', className)}>
```
Comment (AustinHaws-Utah) [inline @utahdts/utah-design-system/react/components/widgetsIndicators/Badge.jsx:16]: Are any aria attributes needed?

Comment (ggrimaudDTS) [reply @utahdts/utah-design-system/react/components/widgetsIndicators/Badge.jsx:16]: Using a visually hidden title instead

File: utah-design-system-website/src/react/components/websiteContent/library/components/widgetsIndicators/badges/BadgesDocumentation.jsx [hunk 5]
```diff
@@ -12,23 +15,63 @@ import {
...
 
 export function BadgesDocumentation() {
+  const { addBanner } = useBanner();
   return (
     <div className="documentation-content">
       <h1 id="h1-top">Badges</h1>
       <p className="lead-in">Badges convey dynamic information, such as counts or status. A badge can include labels or numbers.</p>
       <hr />
       <h2 id="section-example">Example</h2>
+      <SandboxExample
```
Comment (AustinHaws-Utah) [inline utah-design-system-website/src/react/components/websiteContent/library/components/widgetsIndicators/badges/BadgesDocumentation.jsx:35]: When the value is a single digit, like the number "3", the circle is narrow and not a circle. Should it have a min width so it is a circle when the content is small?

File: @utahdts/utah-design-system/react/components/widgetsIndicators/Badge.jsx (hunk 4 above)
Comment (ggrimaudDTS) [reply @utahdts/utah-design-system/react/components/widgetsIndicators/Badge.jsx:16]: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label#sect1

File: utah-design-system-website/src/react/components/websiteContent/library/components/widgetsIndicators/badges/BadgesDocumentation.jsx [hunk 6]
```diff
@@ -145,18 +188,33 @@ export function BadgesDocumentation() {
                     </TableHead>
                     <TableBody>
                       <TableRow>
-                        <TableCell><code className="primary-color">appearance</code></TableCell>
+                        <TableCell><code className="primary-color">children</code></TableCell>
+                        <TableCell><code>react node</code></TableCell>
```
Comment (AustinHaws-Utah) [inline utah-design-system-website/src/react/components/websiteContent/library/components/widgetsIndicators/badges/BadgesDocumentation.jsx:192]: We may want to standardize on what type `children` shows in documentation props. Some say `node`, some say `react node`, some say `React.ReactNode`, or `React Node`.

File: utah-design-system-website/src/react/components/websiteContent/library/components/widgetsIndicators/badges/BadgesExampleProps.jsx [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+ * @param {import('use-immer').Updater<{props: BadgesExamplePropsShape}>} props.setState
+ * @param {object} props.state
+ * @returns {import('react').JSX.Element}
+ */
+
+export function BadgesExampleProps({ setState, state }) {
+  return (
+    <Form
+      className="form--stacked"
+      // @ts-ignore
```
Comment (AustinHaws-Utah) [inline utah-design-system-website/src/react/components/websiteContent/library/components/widgetsIndicators/badges/BadgesExampleProps.jsx:16]: The props.state param has a type of `object`. Change that type to:
``` * @param {{props: BadgesExamplePropsShape}} props.state```
and the @ts-ignore is no longer needed

File: @utahdts/utah-design-system/css/6-components/base-components/widgetsInicators/_badge.scss [hunk 8]
```diff
@@ -0,0 +1,20 @@
+@use "../../../1-settings/color-swatches";
```
Comment (AustinHaws-Utah) [inline @utahdts/utah-design-system/css/6-components/base-components/widgetsInicators/_badge.scss:1]: This filename seems to be missing a 'd':
widgetsInicators -> widgetsIndicators

**Review by AustinHaws-Utah — COMMENTED** [review]
🍱

File: @utahdts/utah-design-system/css/6-components/base-components/widgetsIndicators/_badge.scss [hunk 9]
```diff
@@ -0,0 +1,24 @@
+@use "../../../1-settings/color-swatches";
+@use "../../../1-settings/class-vars";
+@use "../../../1-settings/font-stuff";
+
+#{class-vars.$base-class} {
+  .badge {
```
Comment (ggrimaudDTS) [inline @utahdts/utah-design-system/css/6-components/base-components/widgetsIndicators/_badge.scss:6]: Compare with header badges.

File: utah-design-system-website/src/react/components/websiteContent/library/components/widgetsIndicators/badges/BadgesDocumentation.jsx (hunk 6 above)
Comment (ggrimaudDTS) [reply utah-design-system-website/src/react/components/websiteContent/library/components/widgetsIndicators/badges/BadgesDocumentation.jsx:192]: `React.ReactNode`

**Pull request merged**